umeboshi/utils.py

```diff
@@ -15,13 +15,13 @@
 
 
 @contextlib.contextmanager
-def simple_lock(lock_key, timeout=5000):
+def simple_lock(lock_key, timeout=5000, redis_timeout_sec=5):
```

Comment:
`timeout` is defined but never used after the change. Are we doing it intentionally?

Reply:
yes. if i remember correctly, there's a very similar code in site that does use "timeout" somewhere so I wanted this to be very explicit (that it's a redit lock and it's in seconds, whereas "timeout" is in ms)

Reply:
Oh sorry it's used here 

```
if waited > timeout:
```

Conversation resolved.